nrf_security/configs/nrf-config.h.template

```diff
@@ -3750,7 +3750,12 @@ it is (2^48 - 1), our restriction is :  (int - 0xFFFF - 0xF).*/
 typedef void mbedtls_ecp_restart_ctx;
 #endif
 
-
+/*
+ * Nordic added. Ensure that the MBEDTLS_SSL_SOME_SUITES_USE_TLS_CBC is enabled when PROTO_TLS is enabled.
+ */
+#if defined(MBEDTLS_SSL_PROTO_TLS1) || defined(MBEDTLS_SSL_PROTO_TLS1_1) || defined(MBEDTLS_SSL_PROTO_TLS1_2)
+        #define MBEDTLS_SSL_SOME_SUITES_USE_TLS_CBC
```

Comment:
why is this needed ?
This template ALWAYS defines `MBEDTLS_SSL_PROTO_TLS1_2`: 
https://github.com/nrfconnect/sdk-nrfxlib/blob/8243ed209863ee1a3bf2a5d2735a939e8f890f4f/nrf_security/configs/nrf-config.h.template#L1647

Reply:
sorry, wrong lien, i'm referring to the line above:
```
#if defined(MBEDTLS_SSL_PROTO_TLS1) || defined(MBEDTLS_SSL_PROTO_TLS1_1) || defined(MBEDTLS_SSL_PROTO_TLS1_2)
```

Reply:
Isn't that more future proof if later we want to use TLS1_3?

Reply:
The changes added makes sense, please remove the "request for change" @tejlmand

Reply:
in which way ?
If this is for future proofing if `TLS1_3` support, then the current:
https://github.com/nrfconnect/sdk-nrfxlib/blob/8243ed209863ee1a3bf2a5d2735a939e8f890f4f/nrf_security/configs/nrf-config.h.template#L1647

will probably become a:
```
 #cmakedefine MBEDTLS_SSL_PROTO_TLS1_2 
```
and in that case, the value for `#define MBEDTLS_SSL_SOME_SUITES_USE_TLS_CBC` could be a `cmakedefine` as well, so that it follows the pattern for the rest of the defines in this file.

Reply:
Ah yes. That will be cleaner, agreed...

Reply:
This has been fixed

Reply:
This has been fixed